packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -3255,6 +3255,188 @@ describe("IModelTransformerHub", () => {
     masterSeedDb.close();
   });
 
+  it("should fail older iModels without new versioning behavior unless ignoreNoBranchRelationshipData is true", async () => {
```

Comment:
if I understand correctly
```suggestion
  it("should fail processingChanges on pre-version-tracking forks unless ignoreNoBranchRelationshipData is true", async () => {
```